packages/block-library/src/post-comment-content/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { alignJustify as icon } from '@wordpress/icons';
```

Comment:
Sorry for the nitpick, but let's not use `alignJustify` as icon :sweat_smile: It's just a bit too confusing to see any of the alignment-related icons as the block type icon (it's a really unexpected place).

If you don't wanna use `comment`, we can look for something else that represents written content :thinking:

Reply:
Actually I stole it from Post Content 😅 `comment` works fine if you think `alignJustify` is inappropriate here.

https://github.com/WordPress/gutenberg/blob/beced056e067f5cc34919a31b216491da267a7cf/packages/block-library/src/post-content/index.js#L5

Reply:
> Actually I stole it from Post Content :sweat_smile: 

Oh, I didn't realize! :sweat_smile: 

Okay, while that's not optimal, we can land as-is and defer to designers to get better icons for this.